### PR TITLE
Assignment progress bar

### PR DESCRIPTION
**Pull request by azakon97**

### CODE REVIEW

File: resources/assets/js/components/Resources/ResourcesRow.js [hunk 1]
```diff
@@ -31,7 +31,9 @@ export default class ResourcesRow extends Component {
         <div className="resources-table-cell resources-faces-list">
           {
             this.props.resource.assignments.map((a, j) =>
-              <a href={"/contacts/"+a.student.user.id} alt={a.student.user.name} title={a.student.user.name}><img key={"pp"+j} src={a.student.user.profile_picture} /></a>
+              <a href={"/contacts/"+a.student.user.id} alt={a.student.user.name} title={a.student.user.name}>
+                <img key={"pp"+j} src={a.student.user.profile_picture} />
+              </a>
             )
           }
           <img src="/images/icons8-plus-50.png" onClick={(e) => this.openAddStudent()} />
```
Comment (shravnages) [inline resources/assets/js/components/Resources/ResourcesRow.js]: (e) => this.openAddStudent(e)??

File: resources/assets/js/components/Dashboard.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+
+
+
+
+
+
+if (document.getElementById('resources-widget')) {
+  ReactDOM.render(<Resources />, document.getElementById('resources-widget'));
+}
```
Comment (jasonlipo) [inline resources/assets/js/components/Dashboard.js]: What is this file for? Can it be deleted?

File: resources/assets/js/components/Dashboard/Assignments.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    axios.get('/assignments/list', {
+      _token: $('meta[name="csrf-token"]').attr('content')
+    }).then((response) => {
+      this.setState(response.data);
+      this.changeStyle();
+    });
+  }
+
+  changeStyle() {
+    console.log(this.state.tasks.length);
```
Comment (jasonlipo) [inline resources/assets/js/components/Dashboard/Assignments.js]: Can remove console log for production

File: resources/assets/js/components/Dashboard/Assignments.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
+import React, {Component} from 'react';
+import axios from 'axios';
+import ReactDOM from 'react-dom';
+
+export default class Assignments extends Component {
+  constructor() {
+    super();
+    this.state = {
+      tasks: [],
+      progress_bar_width: 40
```
Comment (jasonlipo) [inline resources/assets/js/components/Dashboard/Assignments.js]: Default to 0? So it doesn't look like the progress bar jumps from 40 to another value

File: resources/assets/js/components/Resources/ResourcesRow.js (hunk 1 above)
Comment (jasonlipo) [reply resources/assets/js/components/Resources/ResourcesRow.js]: @shravnages what does that mean?

**Pull request merged**